src/main/resources/api/service_openapi.yaml

```diff
@@ -158,13 +175,35 @@ components:
       name: offset
       schema:
         type: integer
+        default: 0
       description: The number of entries to skip
     ResultLimit:
       in: query
       name: limit
       schema:
         type: integer
-      description: The numbers of entries to return          
+        format: int64
+      description: The total numbers of entries to return for a query. If ommitted, all entries are returned
+    PageSize:
+      in: query
+      name: pageSize
+      schema:
+        type: integer
+        default: 100
```

Comment:
Actually given these values would it make sense to have a default at the controller level and then you don't need need the constant defaults?